src/main/java/co/elastic/logstash/filters/elasticintegration/EventProcessorBuilder.java

```diff
@@ -316,7 +316,7 @@ public synchronized EventProcessor build(final PluginContext pluginContext) {
     private static ScriptService initScriptService(final Settings settings, final ThreadPool threadPool) throws IOException {
         Map<String, ScriptEngine> engines = new HashMap<>();
         engines.put(PainlessScriptEngine.NAME, getPainlessScriptEngine(settings));
-        engines.put(MustacheScriptEngine.NAME, new MustacheScriptEngine());
+        engines.put(MustacheScriptEngine.NAME, new MustacheScriptEngine(settings));
```

Comment:
review note: addition to spike diff that `MustacheScriptEngine` constructor requires settings.